integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/datasource/CreateTableUsingSparkCarbonFileFormatTestCase.scala

```diff
@@ -254,10 +254,8 @@ class CreateTableUsingSparkCarbonFileFormatTestCase extends FunSuite with Before
 
   private def clearDataMapCache(): Unit = {
     if (!sqlContext.sparkContext.version.startsWith("2.1")) {
-      val mapSize = DataMapStoreManager.getInstance().getAllDataMaps.size()
       DataMapStoreManager.getInstance()
-        .clearDataMaps(AbsoluteTableIdentifier.from(writerPath))
-      assert(mapSize > DataMapStoreManager.getInstance().getAllDataMaps.size())
+        .clearIndex(AbsoluteTableIdentifier.from(writerPath))
```

Comment:
These files are nothing to do this issue right ? please rebase and keep only your issue changes

Reply:
revert changes in this file